### DISCUSSION TITLE
Zxing.Net Mobile.Uno available?

### DISCUSSION THREAD

**Discussion by jp-weber** [unoplatform/uno#16211] [General]
The Toyota blog post mentioned that Zxing.Net.Mobile has been ported to Uno:
https://platform.uno/blog/toyota-migrates-mobile-app-to-uno-platform/

As far as I could see, this is not freely available. Are there any plans to do so?

**Comment by sasakrsmanovic** [comment]
Daniel Causer did a session on using Zxing (and Quagga.JS for web) , and he included some sample code. Does this help? https://platform.uno/blog/using-native-controls-in-uno-platform-applications-demo-code-included/
